akka-docs/src/main/paradox/stream/operators/Flow/asFlowWithContext.md

```diff
@@ -8,12 +8,11 @@ Turns a Flow into a FlowWithContext which can propagate a context per element al
 
 ## Signature
 
-@@signature [Flow.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala) { #asFlowWithContext }
-
+@apidoc[asFlowWithContext](Flow) { scala="#asFlowWithContext[U,CtxU,CtxOut](collapseContext:(U,CtxU)=&gt;In)(extractContext:Out=&gt;CtxOut):akka.stream.scaladsl.FlowWithContext[U,CtxU,Out,CtxOut,Mat]" java="#asFlowWithContext(akka.japi.function.Function2,akka.japi.function.Function)" }
 @@@
```

Comment:
This is the ideal case for the replacement tool: the `asFlowWithContext` operator only has one implementation, the implementation is on the class `Flow` (not a companion object) and the operator has no overloads.